### PR TITLE
Refactor for parameterizing_rule_resolver

### PR DESCRIPTION
**Pull request by ydah**

### CODE REVIEW

File: lib/lrama/grammar.rb [hunk 1]
```diff
@@ -289,6 +293,18 @@ def append_special_symbols
       @accept_symbol = term
     end
 
+    def resolve_inline_rules
```
Comment (yui-knk) [inline lib/lrama/grammar.rb:296]: Is this change intentional? It seems that this method is not called from anywhere, at least `bundle exec rake` successes even so deleting this method.

Comment (ydah) [reply lib/lrama/grammar.rb]: sorry. I updated this PR.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yui-knk) [inline .vscode/settings.json:1]: Could you remove this file from this commit?

**Review by yui-knk — APPROVED** [review]

File: .vscode/settings.json (hunk 2 above)
Comment (ydah) [reply .vscode/settings.json]: 🙏
